enhancements/multi-arch/heterogeneous-architecture.md

```diff
@@ -0,0 +1,348 @@
+---
+title: heterogeneous-architecture-clusters
+authors:
+  - "@Prashanth684"
+reviewers:
+  - "@bparees"
+  - "@derekwaynecarr"
+  - "@jupierce"
+  - "@sdodson"
+  - "@wking"
+  - "@LalatenduMohanty - upgrades"
+  - "@sosiouxme - payload build and release"
+  - "@enxebre - hypershift"
+  - "@dmage - imagestream api changes"
+  - "@kdoberst - console filtering changes"
+  - "@adambkaplan - Openshift Builds"
+  - "@perdasilva - OLM changes"
+  - "@mrunalp - MCO changes"
+approvers:
+  - "@bparees"
+  - "@derekwaynecarr"
+creation-date: 2022-01-17
+last-updated: 2022-01-17
+status: implementable
+---
+
+# heterogeneous-architecture-cluster-support
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Operational readiness criteria is defined
+- [ ] Graduation criteria for tech preview and GA
+- [ ] User-facing documentation is created in [Openshift-docs](https://github.com/Openshift/Openshift-docs/)
+
+## Summary
+
+This enhancement describes the [plan][OCPPLAN-4577] to support provisioning and upgrades of heterogeneous architecture Openshift clusters.
```

Comment:
nit: the OCPPLAN-4577 link, and/or a public analog like [RFE-2240](https://issues.redhat.com/browse/RFE-2240) can go in [the `tracking-link` property][1] in the YAML frontmatter.

[1]: https://github.com/openshift/enhancements/blame/66eb520cb9606170d293432ec3118b04dbfdfa69/guidelines/enhancement_template.md#L15

Reply:
linked. thanks for pointing this out!